kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportConfig.kt

```diff
@@ -0,0 +1,1225 @@
+        }
+
+    @Deprecated(
+        message = "Property was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        replaceWith = ReplaceWith("groupBy"),
+        level = DeprecationLevel.ERROR
+    )
+    public var entity: GroupingEntityType
+        get() {
+            throw KoverDeprecationException("Property entity was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+        set(value) {
+            throw KoverDeprecationException("Property entity was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+
+    @Deprecated(
+        message = "It is forbidden to override filters for a specific report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        level = DeprecationLevel.ERROR
+    )
+    public fun filters(config: Action<KoverReportFiltersConfig>) {
+        throw KoverDeprecationException("It is forbidden to override filters for the verification report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+    }
+
+    /**
+     * Specifies the set of verification rules that control the
+     * coverage conditions required for the verification task to pass.
+     *
+     * An example of bound configuration:
+     * ```
+     * // At least 75% of lines should be covered in order for build to pass
+     * bound {
+     *     aggregationForGroup = AggregationType.COVERED_PERCENTAGE // Default aggregation
+     *     metric = MetricType.LINE
+     *     minValue = 75
+     * }
+     * ```
+     *
+     * @see KoverVerifyBound
+     */
+    public fun bound(config: Action<KoverVerifyBound>)
+
+    /**
+     * A shortcut for
+     * ```
+     * bound {
+     *     min.set(min)
+     * }
+     * ```
+     *
+     * @see bound
+     */
+    public fun minBound(min: Int)
+
+    /**
+     * A shortcut for
+     * ```
+     * bound {
+     *     min.set(min)
+     * }
+     * ```
+     *
+     * @see bound
+     */
+    public fun minBound(min: Provider<Int>)
+
+    /**
+     * A shortcut for
+     * ```
+     * bound {
+     *     max = max
+     * }
+     * ```
+     *
+     * @see bound
+     */
+    public fun maxBound(max: Int)
+
+    /**
+     * A shortcut for
+     * ```
+     * bound {
+     *     max = max
+     * }
+     * ```
+     *
+     * @see bound
+     */
+    public fun maxBound(max: Provider<Int>)
+
+    // Default parameters values supported only in Kotlin.
+
+    /**
+     * A shortcut for
+     * ```
+     * bound {
+     *     minValue = minValue
```

Comment:
This looks obsolete, `minValue` is deprecated and all others are providers/properties now?